dotcom-rendering/src/types/configContext.ts

```diff
@@ -23,3 +24,40 @@ export type Config =
 			assetOrigin: AssetOrigin;
 			editionId: EditionId;
 	  }>;
+
+export const defaultConfig: Config = {
+	renderingTarget: 'Web',
+	darkModeAvailable: false,
+	updateLogoAdPartnerSwitch: false,
+	assetOrigin: 'https://assets.guim.co.uk/',
+	editionId: 'UK',
+};
+
+const assetOrigin = [
+	'https://assets.guim.co.uk/',
+	'https://assets-code.guim.co.uk/',
+	'/',
+] as const;
+
+const editionId = ['UK', 'US', 'AU', 'INT', 'EUR'] as const;
```

Comment:
Could ensure these are valid with a bit more narrowing:

```suggestion
const editionId = ['UK', 'US', 'AU', 'INT', 'EUR'] as const satisfies ReadonlyArray<EditionId>;
```